packages/patternfly-4/react-charts/src/components/Chart/Chart.tsx

```diff
@@ -349,6 +365,7 @@ export const Chart: React.FunctionComponent<ChartProps> = ({
 
   // destructure last
   theme = getTheme(themeColor, themeVariant),
+  containerComponent = allowZoom ? <VictoryZoomContainer /> : <ChartContainer theme={theme} />,
```

Comment:
Can we not pass `desc: ariaDesc, title: ariaTitle,` here and avoid `React.cloneElement(containerComponent as React.ReactElement<any>` later?

Reply:
Don't want to modify a given container component directly. I could add the props here, but would be adding it again later in the code

Reply:
Didn't want to modify a given containerComponent directly; thus, the clone. Could add props to the defaults while destructuring, but still need to add the same props while cloning the given component. Keeping it simple and following Victory, which also clones the containerComponent, labelComponent, etc.